pages/index.page.tsx

```diff
@@ -335,7 +332,7 @@ const Home = (props: any) => {
             <h2 className='text-h3mobile lg:text-h3 text-white mb-6'>
               Start learning JSON Schema
             </h2>
-            <button className='w-[170px] h-[45px] mx-auto rounded border-2 bg-primary text-white font-semibold dark:bg-[560bad] dark:border-none'>
+            <button className='w-[170px] h-[45px] mx-auto hover:bg-blue-700 transition-all duration-300 ease-in-out rounded border-2 bg-primary text-white font-semibold'>
```

Comment:
Here you can apply `dark:bg-transparent`  and `dark:hover:bg-blue-700` class to make the button similar to the above buttons in dark theme.

Reply:
I get what you are trying to suggest but I think having it colored makes it visually easy to identify and in the deployed version also it is colored and not following a skeleton design. So changing that would be out of scope for this particular `enhancement: adding hover effect` and will create confusion in future.